winpr/libwinpr/utils/ntlm.c

```diff
@@ -76,34 +76,15 @@ BOOL NTOWFv1A(LPSTR Password, UINT32 PasswordLength, BYTE* NtHash)
 BOOL NTOWFv2W(LPWSTR Password, UINT32 PasswordLength, LPWSTR User, UINT32 UserLength, LPWSTR Domain,
               UINT32 DomainLength, BYTE* NtHash)
 {
-	BYTE* buffer;
 	BYTE NtHashV1[16];
```

Comment:
Maybe a good idea to replace the magic number with the `WINPR_MD5_DIGEST_LENGTH` macro